<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>expWin</class>
 <widget class="QWidget" name="expWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="0" colspan="2">
    <widget class="QPushButton" name="refillButton">
     <property name="text">
      <string>Refill</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="changeMouseRespButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Stimulus Probability:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLineEdit" name="mouseRespLineEdit"/>
   </item>
   <item row="8" column="0" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="liquidLineEdit"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Trial limit:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_blah">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Change parameters for all mice</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Change Liquid Amount:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLineEdit" name="trialLimLineEdit"/>
   </item>
   <item row="9" column="4">
    <widget class="QPushButton" name="changeTrialLimButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="refillLabel">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Valve is closed</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="pauseLabel">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Experiment is ongoing</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QPushButton" name="pauseButton">
     <property name="text">
      <string>Pause Experiment</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="trial_lim_lab">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mouse limit per day</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="changeMouseLimButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="changewaittimeButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="waittimeLineEdit"/>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Change Response Time (ms):</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="mouseLimLineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="changeliquidButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QPushButton" name="changeStimProbButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Change Lick Threshold:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLineEdit" name="stimProbLineEdit"/>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="changelickButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Change Wait Time (ms):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lickLineEdit"/>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="punishtimeLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Change Punish Time (ms):</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="changepunishtimeButton">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
